content/wireless.md

```diff
@@ -16,34 +16,164 @@ section: network-troubleshooting
 tableOfContents: true
 ---
 
-WiFi issues are influenced by many different factors, including:
+If your computer can’t connect to wireless networks or the connection is unstable, use this guide to diagnose and resolve the issue.  
+Start with the quick checks, then follow the targeted diagnostic and recovery steps.  
+Include logs when you ask for support.
 
-- Hardware (WiFi card, access point)
-- Settings at both ends of the connection
-- The local environment
+---
+
+## Quick Checks (First Steps)
+
+Try these steps before diving into advanced troubleshooting:
+
+- Reboot the router/modem and the computer.
+- Toggle Airplane Mode: **Fn + F11 → wait 10s → disable Airplane Mode.**
+- If your Wi-Fi issues started after an update, try removing the backported Wi-Fi driver:
+
+```bash
+  sudo apt remove backport-iwlwifi-dkms
+```
+Then restart your computer.
+Temporarily use a phone hotspot to confirm whether the network or your computer is the issue.
+Boot from a Live USB of your distribution to determine whether the issue exists outside your installed system.
```

Comment:
5. [Boot from a Live USB](https://support.system76.com/articles/live-disk) of your distribution to determine whether the issue exists outside your installed system.